.github/actions/install-chainweb-from-artifacts/action.yml

```
@@ -17,10 +17,10 @@ runs:
       run: |
         case "${{ inputs.os }}" in
         ubuntu*)
-          sudo apt-get install -y librocksdb-dev z3
+          sudo apt-get install -y z3 build-essential 
```

I think, z3 can be dropped here as well, but not 100% sure.
```suggestion
          sudo apt-get install -y build-essential 
```